cmd/minikube/cmd/start.go

```diff
@@ -88,6 +88,9 @@ type versionJSON struct {
 	Commit          string `json:"commit"`
 }
 
+// ErrPatchNotFound is when given volume was not found
+var ErrPatchNotFound = errors.New("Unable to detect the latest patch release for specified major.minor version")
```

Comment:
this error message doesn't mention it is talking about Kubernetes version, that could be ambiguous to the user.

how about something like this
ErrKubernetesPatchNotFound = errors.New("Unable to detect the latest patch release for specified Kubernetes version")